### PR TITLE
PEP8 + handle post requests

### PR DESCRIPTION
**Pull request by Phyks**
This PR fixes PEP8 compliance and allow the user to use a post request.

To do so, set the according setting to "post" (instead of "get") and edit the url with a correct one. You may have to change the post field name on line 143.

Tested and approved with https://github.com/Phyks/shortURL using the base url with the "?api=1" param (for instance "http://u.phyks.me/?api=1").

### CODE REVIEW

File: urlshortener.py [hunk 1]
```diff
@@ -42,13 +42,12 @@
     'replace_out.status': 'off',
 
     'length': '45',
-    'shortener': 'http://is.gd/create.php?format=simple&url=%s'
+    'shortener': 'http://is.gd/create.php?format=simple&url=%s',
+    'shortener_type': 'get'
```
Comment (Niols) [inline urlshortener.py]: Wouldn't we want to call this `method`?

File: urlshortener.py [hunk 2]
```diff
@@ -59,13 +58,14 @@
 )
```
Comment (Niols) [inline urlshortener.py:60]: If you have something to build a better regexp for URLs (forgetting files, localhost, …), feel free to commit it!

File: urlshortener.py (hunk 1 above)
Comment (Phyks) [inline urlshortener.py]: True.

File: urlshortener.py (hunk 2 above)
Comment (Phyks) [inline urlshortener.py:60]: I will check it.

**Pull request merged**